gourmet/backends/db.py

```diff
@@ -894,28 +894,33 @@ def get_unique_values (self, colname,table=None,**criteria):
                   ]
         return [x for x in retval if x is not None] # Don't return null values
 
-    def get_ingkeys_with_count (self, search={}):
+    def get_ingkeys_with_count (self, search=None):
```

Comment:
since you touch the header, please add proper type annotations, of what we expect.

Reply:
I guess it's `Optional[Mapping[str, Any]]` for the input which is quite bad. Maybe you can add a ticket to extract the search-Object into some namedTuple or dataclass, as we only access a few known fieldnames. That's not a proper Mapping type at all.

The return type is probably something like `List[Tuple[str, int]]`, right? Why is this not a dict anyway?

Reply:
Created issue #96